### PR TITLE
Cafcass & Pathfinder WA Task changes

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-6155
https://tools.hmcts.net/jira/browse/PRL-5951
https://tools.hmcts.net/jira/browse/PRL-6191
https://tools.hmcts.net/jira/browse/PRL-6262

### Change description ###
WA path finder , close, reopen case changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by shashisk97 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java [hunk 1]
```diff
@@ -3276,12 +3281,28 @@ public Map<String, Object> setFieldsForWaTask(String authorisation, CaseData cas
...
+                || eventId.equals(ADMIN_EDIT_AND_APPROVE_ORDER.getId()))
+            && Yes.equals(caseData.getServeOrderData().getCafcassOrCymruNeedToProvideReport())
+            && ObjectUtils.isNotEmpty(caseData.getServeOrderData().getWhenReportsMustBeFiled())
+            && DAYS.between(LocalDate.now(), caseData.getServeOrderData().getWhenReportsMustBeFiled()) >= 7
+            && YesOrNo.Yes.equals(caseData.getIsPathfinderCase())) {
+            log.info("Inside setFieldsForRequestSafeGuardingReportWaTask");
+            waFieldsMap.put(WA_REQ_SER_UPDATE, "Yes");
+            waFieldsMap.put(
+                WA_SER_DUE_DATE, caseData.getServeOrderData().getWhenReportsMustBeFiled().minusDays(6).format(DateTimeFormatter.ISO_LOCAL_DATE));
+            log.info("waFieldsMap => " + waFieldsMap);
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java:3302]: remove this logger after testing @sahaSonali

File: src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java [hunk 2]
```diff
@@ -37,8 +47,8 @@ public class CafcassUploadDocService {
 
     public static final List<String> ALLOWED_FILE_TYPES = List.of("pdf", "docx");
     public static final List<String> ALLOWED_TYPE_OF_DOCS = List.of("16_4_Report", "CIR_Part1", "CIR_Part2", "CIR_Review", "CMO_report",
-        "Contact_Centre_Recordings", "Correspondence", "Direct_work", "Enforcement_report", "Enquiry_from_Foreign_Court", "FAO_Report",
-        "FAO_Workplan", "High_Court_Team_Template", "Letter_from_Child", "Other_Non_Section_7_Report", "Parental_Order_Report", "Position_Statement",
+        "Contact_Centre_Recordings", "Correspondence", "Direct_work", "Enforcement_report", "FAO_Report",
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java:50]: Why is this list changed?

**Review by shashisk97 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java (hunk 2 above)
Comment (sahaSonali) [reply src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java:50]: As per the requirement we have changed it

**Review by shashisk97 — APPROVED** [review]

**Pull request merged**